packages/falso/src/lib/code-snippet.ts

```diff
@@ -0,0 +1,56 @@
+import { codeSnippetLangToCodeSnippet } from './code-snippet-data';
+import { fake, FakeOptions } from './core/core';
+
+export type CodeSnippetLang =
```

Comment:
Can you please leave only the most common:

  | 'bash'
  | 'c'
  | 'c#'
  | 'c++'
  | 'css'
  | 'go'
  | 'html'
  | 'java'
  | 'javascript'
  | 'php'
  | 'python'
  | 'rust'
  | 'sql'
  | 'swift';

Reply:
done